open-company/open-company-web
# Archived topics

card: card: https://trello.com/c/P9pykwGj

#### Please merge after #124 or merge only this after reviewing both

To test:
- archive a topic
- check the add topic popover to see if it appears in the archived topics section
- archive some more topics
- check the order of the topics: they should be in alphabetical order
- add back a topic: you should see the topic with the old content
- merge
